src/Component/Avatar/Avatar.tsx

```diff
@@ -22,7 +30,7 @@ export const Avatar: React.FC<ImgProps> = (
 ) => {
     const { imgsrc,order } = props;
     return (
-        <ImgComp order={order}><img style={{ height: AvatarImgSize.height, width: AvatarImgSize.width, display: "block", margin: "auto" }} src={imgsrc} alt="bot" /></ImgComp>
+        <ImgComp order={order} data-testid="AvatarComponent"><Img src={imgsrc} data-testid="AvatarImage"/></ImgComp>
```

Comment:
```suggestion
        <ImgComp order={order} data-testid="Avatar__image-wrapper"><Img src={imgsrc} data-testid="Avatar__image"/></ImgComp>
```
This can be used for cypress tests as well so we should prefix the id's with component names.